omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1606 @@
+      - Varies†
+      - Varies†
+      - N/A
+      - Varies†
+    * - 4.3
+      - Varies†
+      - Varies†
+      - Varies†
+      - Varies†
+      - Varies†
+      - N/A
+      - Varies†
+    * - 4.4
+      - Varies†
+      - Varies†
+      - Varies†
+      - Varies†
+      - Varies†
+      - N/A
+      - Varies†
+    * - 4.5
+      - Varies†
+      - Varies†
+      - Varies†
+      - Varies†
+      - Varies†
+      - N/A
+      - Varies†
+    * - Details
+      - 
+      - 
+      - 
+      - 
+      - 
+      - `Link <https://developer.apple.com/graphicsimaging/opengl/capabilities/index.html>`__
+      - 
+
+\*
+  Driver/hardware dependent, but the Windows Vista/7/8 DirectX 9
+  requirement is mostly the same as OpenGL 2.0, so any badged machine
+  should have OpenGL 2.0-capable hardware (it will also require a
+  functional driver)
+
+†
+  Driver/hardware dependent
+
+Support is largely down to the system hardware (GPU) and drivers.  No
+guarantees for 2.0 and greater on Windows (1.4 only guaranteed), but
+AMD, Intel and nVidia all provide GL drivers supporting newer
+versions.  The minimum baseline is probably 2.1.  However, 3.2/3.3 is
+probably the baseline for most hardware under 5 years old, with 4.x
+being the typical baseline for hardware from a year back.
+
+Apache
+^^^^^^
+
+`General overview <http://www.apachehaus.com/index.php?option=com_content&view=article&id=119&Itemid=104>`__ and `website <http://httpd.apache.org/>`__.
+
+.. tabularcolumns:: |l|l|l|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - OME (build)
+      - OME (deploy)
+    * - 2.0
+      - from Nov 2001
+      - to Jul2013
+      - N/A
+      - Dropped
+    * - 2.2
+      - from Dec 2005
+      - TBA
+      - N/A
+      - Supported
+    * - 2.4
+      - from Feb 2012
+      - TBA
+      - N/A
+      - Recommended
+
+Distribution support:
+
+.. tabularcolumns:: |l|l|L|L|L|l|l|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - CentOS/RHEL
+      - Fedora
+      - Ubuntu
+      - Debian
+      - Homebrew
+      - FreeBSD Ports
+    * - 2.0
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+    * - 2.2
+      - 6.x
+      - N/A
+      - 10.04, 12.04, 13.10
+      - 6.0, 7.0
+      - N/A
+      - Yes
+    * - 2.4
+      - 7.x
+      - 19, 20, 21, (22)
+      - 13.10, 14.04, 14.10, (15.04)
+      - (8.0)
+      - N/A
+      - Yes
+    * - Details
+      - 
+      - `Link <https://apps.fedoraproject.org/packages/httpd>`__
+      - `Link <http://packages.ubuntu.com/search?keywords=apache&searchon=names&suite=all&section=all>`__
+      - `Link <https://packages.debian.org/search?keywords=apache&searchon=names&suite=all&section=all>`__
+      - 
+      - 
+
+Apache 2.2 requires mod_fastcgi (from external repository, for
+CentOS6).  Apache 2.4 requires mod_proxy_cfgi (`to be tested
+<https://trello.com/c/ROD6R0L7/180-omero-web-custom-prefix>`__).
```

Comment:
This is another thing to discuss.  If it needs testing internally, we need to do so then we can update this with the proper information.

@manics What's the status of this one?